data/json.go

```diff
@@ -1052,6 +1150,51 @@ func (j *jsonConnector) SearchMovieTitles(query string) ([]*common.Movie, error)
 	return found, nil
 }
 
+// This function filters the given movies by the supplied tags
+// To be returned a movie has to match ALL supplied tags
+func (j *jsonConnector) FilterMoviesByTags(movies []*common.Movie, tags []string) ([]*common.Movie, error) {
```

Comment:
I'm not sure this function belongs in the connector.  Movies are passed in and so are the tags.  This isn't something that's specific to the JSON implementation.

A better place would be in `common.Movie.go`.

Reply:
Agreed, an other oversight

Reply:
moved to common/util.go